index.ts

```diff
@@ -108,6 +109,21 @@ export const isSidePanel = once((): boolean => {
 	return url.pathname === location.pathname;
 });
 
+export const isActionPopup = once((): boolean => {
+	// Chrome-only; Firefox uses the whole window…
+	if (globalThis.outerHeight - globalThis.innerHeight === 14) {
```

Comment:
Magic number targeting Chrome. On Firefox the check only works if the URL matches the default URL (as is the case for most other checks)